remote/remote.go

```diff
@@ -49,6 +50,24 @@ type Remote interface {
 
 	// GetHook gets the hook from the http Request.
 	GetHook(r *http.Request) (*model.Hook, error)
+
+	// GetStatusHook gets the status hook from the http Request.
+	GetStatusHook(r *http.Request) (*model.StatusHook, error)
+
+	// GetBranchStatus returns overall status for the named branch from the remote system
+	GetBranchStatus(*model.User, *model.Repo, string) (*model.BranchStatus, error)
+
```

Comment:
Is this an unused function? I did a quick ctrl+f search for `GetBranchStatus` but didn't see where it was being invoked. Sorry if I'm missing